Automattic/wp-calypso
# Restore Jetpack Earn Upsells and Settings for new Plans

#### Changes proposed in this Pull Request

* Change copy on `/earn` to use new Jetpack plan names
* replace upsell on `/earn/ads-earnings`

![Untitled 4@0 5x](https://user-images.githubusercontent.com/2810519/94069678-d7ca1e00-fda5-11ea-97ff-ff9548d94090.png)

<img width="1097" alt="Screen Shot 2020-09-21 at 1 10 19 PM" src="https://user-images.githubusercontent.com/2810519/93816051-cfd67680-fc0b-11ea-9a0f-338209b8caec.png">

#### Testing instructions

Test above routes with Free Jetpack plans and paid ( security daily will do ) 

1. Navigate to `/earn` with a WordPress.com Free, Jetpack Free, and new Jetpack Paid plan
2. Make sure the screens above match ( pat special attention to copy )
3. Go `/earn/ads-earnings` with a Jetpack Free Plan
4. Make sure the "upgrade" button takes you straight to the cart with security daily plan
